app/assets/javascripts/workarea/storefront/paypal/modules/paypal_buttons.js

```diff
@@ -57,9 +57,17 @@ WORKAREA.registerModule('paypalButtons', (function () {
          * @memberof WORKAREA.paypalButtons
          */
         init = function ($scope) {
-            var $buttonContainer = $('#paypal-button-container', $scope);
+            var $buttonContainer = $('#paypal-button-container', $scope),
+                paypalNotLoaded = window.paypal === undefined,
+                paypalExpectedToLoad = !_.isEmpty(
```

Comment:
According to the `include_paypal_javascript_tag` helper method, the `<script>` tag won't even render if the gateway is not configured.